docs/ProfilerOptions.md

```diff
@@ -0,0 +1,113 @@
+| `-g --sig`     | `sig`           | Print method signatures.                                                                                                                                                                                                                                                                                                                                                                                                                             |
+| `-l --lib`     | `lib`           | Prepend library names to symbols, e.g. ``libjvm.so`JVM_DefineClassWithSource``.                                                                                                                                                                                                                                                                                                                                                                      |
+| `--total`      | `total`         | Count the total value of the collected metric instead of the number of samples, e.g. total allocation size.                                                                                                                                                                                                                                                                                                                                          |
+
+## Options not applicable JFR, FlameGraph and Tree view outputs
+| asprof     | Launch as agent | Description                                                                                                                                                                                                                                                                                                                                                                                                                                          |
+|------------|-----------------|------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| `-a --ann` | `ann`           | Annotate JIT compiled methods with `_[j]`, inlined methods with `_[i]`, interpreted methods with `_[0]` and C1 compiled methods with `_[1]`.                                                                                                                                                                                                                                                                                                         |
```

Comment:
Can be merged into the above table (non-JFR) for simplicity/readability. However, in the description we may add that FlameGraph and Tree view will color frames depending on their type regardless of this option.

Reply:
Addressed in [e6282c8](https://github.com/async-profiler/async-profiler/pull/1029/commits/e6282c83bc0aa6007df310fd5e638e9d4d5ed6f1)